modules/store/src/selector.ts

```diff
@@ -120,81 +120,122 @@ export function defaultMemoize(
 export function createSelector<State, S1, Result>(
   s1: Selector<State, S1>,
   projector: (s1: S1) => Result
-): MemoizedSelector<State, Result>;
+): MemoizedSelector<State, Result, (s1: S1) => Result>;
```

Comment:
I think this can be replaced with `typeof projector` in all of the function definitions.

Reply:
Nice! Thanks.